### PR TITLE
Update sending-emails.md

### PR DESCRIPTION
**Pull request by justinkuntz**
Seems obvious but figured for some it might not be. Added `import { Prisma } from '@prisma/client'` for users service. 

Nice little cookbook...I had SendInBlue running in no time!

### CODE REVIEW

**Comment by jtoar** [comment]
Hey @justinkuntz, it's actually not obvious to me either haha. Do you need to import this for the type you mean?

**Comment by justinkuntz** [comment]
image


Prior to import, there was a namespace issue (see screenshot). It looks like Prisma has some extra goods to make things type-safe thus clearing the dreaded red squiggles :) : https://www.prisma.io/docs/concepts/components/prisma-client/advanced-type-safety

File: docs/docs/how-to/sending-emails.md [hunk 1]
```diff
@@ -226,6 +226,10 @@ And then in the users service we'll just create a dummy method to start with.
 ```ts title="users.ts"
 // ...
 
+import { Prisma } from '@prisma/client'
```
Comment (jtoar) [inline docs/docs/how-to/sending-emails.md:229]:
```suggestion
import type { Prisma } from '@prisma/client'
```

**Review by jtoar — COMMENTED** [review]
@justinkuntz thanks for your patience, just got around to testing this one on a real project. Definitely works as is, but I have a preference for `import type` if it's only imported for TS. Just makes the code a little easier to read. Thoughts?

File: docs/docs/how-to/sending-emails.md (hunk 1 above)
Comment (justinkuntz) [reply docs/docs/how-to/sending-emails.md:229]: Makes sense 👍🏼

**Review by jtoar — APPROVED** [review]

**Pull request merged**